mapstruct/src/test/java/com/baeldung/mapper/unmapped/DocumentMapperWithConfigUnitTest.java

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.mapper.unmapped;
+
+import com.baeldung.dto.DocumentDTO;
+import com.baeldung.entity.Document;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
You're mixing JUnit 5 tests with JUnit 4 assertions... I think that's a bit tricky to follow.